plugin/build.gradle

```diff
@@ -131,7 +118,7 @@ afterEvaluate {
         fileMode 0644
         dirMode 0755
 
-        requires('elasticsearch-oss', versions.elasticsearch, EQUAL)
```

Comment:
Not sure where the `versions.elasticsearch` is defined and if it all it's being used. It'll be great if someone can point the reference.